ado/ipacheckreplace.ado

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Ah, that was a mistake! I have it on another branch, so deleting it won't be a problem.